pkg/webui/console/containers/devices-table/index.js

```diff
@@ -146,18 +170,32 @@ class DevicesTable extends React.Component {
     super(props)
 
     this.getDevicesList = filters =>
-      getDevicesList(props.appId, filters, ['name'], { withLastSeen: true })
+      getDevicesList(
+        props.appId,
+        filters,
+        ['name', 'application_server_address', 'network_server_address', 'join_server_address'],
+        { withLastSeen: true, withStatus: true },
+      )
   }
 
   @bind
   baseDataSelector(state) {
-    const { mayCreateDevices, appId } = this.props
+    const { mayCreateDevices, appId, asConfig } = this.props
     const devices = selectDevices(state)
     const decoratedDevices = []
+    const { base_url: stackAsUrl } = asConfig
+    const currentHost = getHostnameFromUrl(stackAsUrl)
+
     for (const device of devices) {
       decoratedDevices.push({
         ...device,
-        _derivedLastSeen: selectDeviceDerivedLastSeen(state, appId, device.ids.device_id),
+        status: {
+          _derivedLastSeen: selectDeviceDerivedLastSeen(state, appId, device.ids.device_id),
+          otherCluster:
```

Comment:
Here it makes sense to create a new selector `isOtherClusterDevice` and implement this logic within that selector so you can reuse in the end device overview